emacs/ocamlformat.el

```diff
@@ -157,17 +157,26 @@ function."
               (error "invalid rcs patch or internal error in ocamlformat--apply-rcs-patch")))))))))
 
 (defun ocamlformat--process-errors (filename tmpfile errorfile errbuf)
-  (with-current-buffer errbuf
-    (if (eq ocamlformat-show-errors 'echo)
-      (message "%s" (buffer-string))
-      (insert-file-contents errorfile nil nil nil)
-      ;; Convert the ocamlformat stderr to something understood by the compilation mode.
-      (goto-char (point-min))
-      (insert "ocamlformat errors:\n")
-      (while (search-forward-regexp (regexp-quote tmpfile) nil t)
-        (replace-match (file-name-nondirectory filename)))
-      (compilation-mode)
-      (display-buffer errbuf))))
+  (if errbuf
+    (progn
+      (with-current-buffer errbuf
+        (setq buffer-read-only nil)
+        (erase-buffer))
+      (with-current-buffer errbuf
+        (if (eq ocamlformat-show-errors 'echo)
+          (message "%s" (buffer-string))
+          ;; Checks the size of errorfile
+          (if (> (nth 7 (file-attributes errorfile)) 0)
```

Comment:
```suggestion
          (if (> (file-attribute-size (file-attributes errorfile)) 0)
```
This works for me :)

Reply:
I added a comment to keep track of why it's not the case